HW34/src/main/java/org/my/homework/ProductRepository.java

```diff
@@ -0,0 +1,27 @@
+package org.my.homework;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductRepository {
+
+    @Bean
+    public List<Product> getAllProductsList() {
```

Comment:
Oh no, this is not a bean... 
The ProductRepository is a spring component (in other words, a bean). This component manages products (something like Facad, which hides the persistence implementation). So, there is no sense making List of Products a bean - ProductRepository is responsible for it.

Reply:
Not sure that a new realisation is much better but here was fixed